compact/hcal/barrel_gdml.xml

```diff
@@ -15,11 +15,11 @@
           readout="HcalBarrelHits"
           vis="HcalBarrelVis"
           env_vis="HcalBarrelEnvelopeVis"
-          rmin1="HcalBarrel_rmin"
-          rmin2="ForwardServiceGap_rmax - 5.0"
-          rmax="HcalBarrel_rmax"
-          z1="SolenoidBarrel_length"
-          z2="HcalBarrel_length"
+          rmin1="HcalBarrel_rmin1 - 2.0"
+          rmin2="HcalBarrel_rmin2 - 2.0"
+          rmax="HcalBarrel_rmax + 2.0"
+          z1="HcalBarrel_length1 + 0.01"
+          z2="HcalBarrel_length2 + 0.01"
```

Comment:
Are these 'fudge factors' necessary? If so, does that mean that the gdml objects are extending outside the envelope?

Reply:
The lengths in the detector table are exactly the lengths of the sectors. Without the fudge factors I had a few tens of micron overlaps between the sector components and the mother volume.  The radial dimensions I didn't look very carefully, that could likely be tighter.

Reply:
Latest commit removes length fudge factors after a first pass on updating the parameter tables with Tanja and Walt. I have tightened the fudge factors for the radius but there is still a disagreement for the inner radii and the length of the central section (the definition of which is ambiguous).  Changing this to a draft for now until I can resolve remaining issues with Tanja and Walt.

Reply:
Can we merge the parameters that have no issues and leave the inner radii and length of the central section to another PR? In this case we can break the big issue (parameter update for HCal) into smaller ones.

Reply:
I think we have this close to resolved. I sent Tanja and Walt a set of updated numbers taken directly from the 3D model just yesterday. The problem was that they are working ONLY from a 2D model that has an incorrect model of the barrel HCAL.  You can tell just by looking at it because the plates show flat in the cross section and they are actually tilted.  

I just need to find some time to cross-check what is in the simulations, which I can hopefully find time to do this afternoon. 

[HCAL_Geometry_Summary_from_3D.pdf
[EPIC Envelope - 09-14-2023.pdf](https://github.com/eic/epic/files/13500724/EPIC.Envelope.-.09-14-2023.pdf)
](https://github.com/eic/epic/files/13500723/HCAL_Geometry_Summary_from_3D.pdf)